crates/storage/src/schema/revision_0058.rs

```diff
@@ -0,0 +1,21 @@
+use anyhow::Context;
+
+pub(crate) fn migrate(tx: &rusqlite::Transaction<'_>) -> anyhow::Result<()> {
+    tracing::info!(
+        "Removing count columns, and adding state diff commitment and length columns to \
+         block_headers table"
+    );
+
+    tx.execute_batch(
+        r"ALTER TABLE block_headers ADD COLUMN state_diff_length INTEGER DEFAULT 0;
+        ALTER TABLE block_headers ADD COLUMN state_diff_commitment BLOB;
+        ALTER TABLE block_headers DROP COLUMN storage_diffs_count;
+        ALTER TABLE block_headers DROP COLUMN nonce_updates_count;
+        ALTER TABLE block_headers DROP COLUMN declared_classes_count;
+        ALTER TABLE block_headers DROP COLUMN deployed_contracts_count;",
```

Comment:
We could technically keep the state diff length I think? Or is that pointless since we don't have the state diff commitment anyway?

Reply:
This is exactly what I initially did. I've just removed the (length) update statement because of the missing commitment in order to be consistent.